day_7/10_speckled_frogs.js

```diff
@@ -0,0 +1,46 @@
+
+var words = ["Ten", "Nine", "Eight", "Seven", "Six",
+             "Five", "Four", "Three", "Two", "One"];
+
+var wordsLength = words.length;
+
+function speckledFrogs(frogs) {
+  for (i = frogs; i >= 10 ; i--) {
```

Comment:
Good work using iteration and conditionals. There'a lots of repeated code, which is an opportunity to think through how you might reduce repetition.